src/WebUI/WorkLoads/WorkloadsPivot.tsx

```diff
@@ -118,6 +120,7 @@ export class WorkloadsPivot extends BaseComponent<IWorkloadsPivotProps, IWorkloa
         return (<DeploymentsTable
```

Comment:
as discussed we should have singleton, so we do not need to send this service around, same with kubeservice also.